scripts/build.sh

```diff
@@ -270,27 +276,19 @@ function publish_package()
             $TP_ROOT_DIR/src/datacollector/datacollector.csproj
         )
 
-        if [ "$framework" == "net451" ] && ! $TPB_HasMono; then
-            # Skip publish if mono is not available
-            continue
-        fi
-
         log "Package: Publish projects for $framework"
         for project in "${projects[@]}" ;
         do
             log ".. Package: Publish $project"
             $dotnet publish $project --configuration $TPB_Configuration --framework $framework --output $packageDir -v:minimal -p:Version=$TPB_Version -p:CIBuild=$TPB_CIBuild -p:LocalizedBuild=$TPB_LocalizedBuild
         done
 
-        # Copy TestHost for desktop targets if we've built net451
-        # packages with mono
-        if $TPB_HasMono; then
-            local testhost=$packageDir/TestHost
-            mkdir -p $testhost
-            cp -r src/testhost/bin/$TPB_Configuration/net451/win7-x64/* $testhost
-            cp -r src/testhost.x86/bin/$TPB_Configuration/net451/win7-x64/* $testhost
-        fi
-        
+        # Copy TestHost for desktop targets
+        local testhost=$packageDir/TestHost
```

Comment:
Can we publish testhost here as we doing in build.ps1.?

Reply:
I will take this up in another PR.